lib/gis/strings.c

```diff
@@ -48,6 +49,39 @@ int G_strcasecmp(const char *x, const char *y)
     return _strncasecmp(x, y, -1);
 }
 
+/*!
```

Comment:
Please move this function to the top of the file and switch the order of the prototypes above.

Reply:
How about place it at the bottom of the file, above impl. of `_strncasecmp`? The other static func.

Reply:
That makes more sense. Static functions together, G_* first. Just noticed that the implementations of both functions lack `static`. Please add it.